src/fhirtypes/ElementDefinition.ts

```diff
@@ -989,6 +991,12 @@ export class ElementDefinition {
       this.fixedMarkdown = value;
     } else if (type === 'uuid' && this.checkIfFixable(value, this.fixedUuid, type)) {
       this.fixedUuid = value;
+    } else if (
+      type == 'xhtml' &&
+      sax.parser(true).write(value).error == null &&
```

Comment:
If I'm reading the source code for `sax` correctly, `write()` has several scenarios in which it throws.  I don't know if we'd ever encounter them, but should we try to handle thrown errors in case they occur?

Reply:
Ah, I didn't catch that when I glanced at the docs. With the new `checkXhtml` helper method, if it happens to throw an exception, the input value will be considered invalid.